templates/decision/rhdm713-authoring-ha.yaml

```
@@ -4,18 +4,18 @@ metadata:
   annotations:
     iconClass: icon-jboss
     tags: rhdm,decision,jboss,authoring,ha
-    version: "7.13"
-    openshift.io/display-name: Red Hat Process Automation Manager 7.13 authoring environment (HA, persistent, with https) for decisions.
+    version: "8.0"
+    openshift.io/display-name: Red Hat Process Automation Manager 8.0 authoring environment (HA, persistent, with https) for decisions.
```

```suggestion
    openshift.io/display-name: IBM Business Automation Manager Open Editions 8.0 authoring environment (HA, persistent, with https) for decisions.
```
Should this be changed here? there are other occurences too

---

Templates will be deleted.